.github/workflows/release.yml

```diff
@@ -124,7 +133,7 @@ jobs:
       - name: get assets
         uses: actions/download-artifact@v4
         with:
-          pattern: qjs-*
+          pattern: *
```

Comment:
@saghul is this safe? I really have no idea what I'm doing here.

Reply:
It should be fine since we are doing the upload.

Reply:
Fingers crossed then!

Reply:
Apparently a plain asterisk is invalid syntax? I'm getting CI notifications:

> You have an error in your yaml syntax on line 136

Um... suggestions?

Reply:
Maybe run the step twice with 2 separate patterns?